platform/main.roc

```diff
@@ -20,12 +23,74 @@ platform "webserver"
     packages {}
     imports [
         Task.{ Task },
+        Stderr.{ line },
     ]
-    provides [mainForHost]
+    provides [forHost]
```

Comment:
I changed it to `forHost` here because I think it is useful to be distinct from the other platforms that have only one `main` entry point, and clarify that this is what roc is providing for the host. Hopefully as this will be an example platform people look at for ideas, we can move away from only having "mainForHost" in all of the platforms.